### PR TITLE
Update for RPM plugin dir location change

### PR DESCRIPTION
**Pull request by findepi**
After this change, presto-admin cannot install a jar in old Presto versions.

Relates to https://github.com/prestosql/presto/pull/2059

### CODE REVIEW

File: docs/presto-admin-commands.rst [hunk 1]
```diff
@@ -198,7 +198,6 @@ configurations:
     node.launcher-log-file=/var/log/presto/launcher.log
     node.server-log-file=/var/log/presto/server.log
     catalog.config-dir=/etc/presto/catalog
-    plugin.dir=/usr/lib/presto/lib/plugin
```
Comment (mosabua) [inline docs/presto-admin-commands.rst]: you sure you can omit that instead of explicitly defining it .. the default value might not work..

Comment (findepi) [reply docs/presto-admin-commands.rst]: Why do you think so?

https://github.com/prestosql/presto/blob/fca01af7715afd531261343b43dd88d3642e56d6/presto-main/src/main/java/io/prestosql/server/PluginManagerConfig.java#L28

https://github.com/airlift/airlift/blob/6d3d6c968d97415320cf44b646dea4de382af9ba/launcher/src/main/scripts/bin/launcher.py#L176-L179

Comment (mosabua) [reply docs/presto-admin-commands.rst]: Yeah .. sorry.. commented to early..

Comment (mosabua) [reply docs/presto-admin-commands.rst]: but something must still be wrong since build fails .. I did not look in detail though

**Comment by anusudarsan** [comment]
@findepi superseded by https://github.com/prestosql/presto-admin/pull/15

**Comment by mosabua** [comment]
This should be closed since it is replaced by https://github.com/prestosql/presto-admin/pull/15

**Pull request closed without merging**